sdk/apimanagement/Azure.ResourceManager.ApiManagement/tests/ApiManagementManagementTestBase.cs

```diff
@@ -25,17 +27,26 @@ protected ApiManagementManagementTestBase(bool isAsync)
         }
 
         [SetUp]
-        public void CreateCommonClient()
+        public async Task CreateCommonClient()
         {
             Client = GetArmClient();
+            DefaultSubscription = await Client.GetDefaultSubscriptionAsync().ConfigureAwait(false);
```

Comment:
Please also consider we make this change in the template as well if this is usually needed in most RPs

Reply:
Done